### PR TITLE
Book, don't run rust code snippets and update getting started

### PR DESCRIPTION
**Pull request by g-re-g**
A typo fix and some small suggestions to the book.

Disables running of rust examples. They don't compile and the error output makes the page cluttery. The button is right next to the `copy` button and is easy to click:
Screenshot 2023-07-13 at 5 24 11 PM

### CODE REVIEW

File: docs/book/src/02_getting_started.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ If you don’t already have it installed, you can install Trunk by running
 cargo install trunk
 ```
 
-Create a basic Rust binary project
+Create a basic Rust project
```
Comment (g-re-g) [inline docs/book/src/02_getting_started.md:17]: I'm not sure the word `binary` is helping folks who want to build Leptos apps and binary is the default these days.

File: docs/book/src/02_getting_started.md [hunk 2]
```diff
@@ -23,7 +23,12 @@ cargo init leptos-tutorial
 `cd` into your new `leptos-tutorial` project and add `leptos` as a dependency
 
 ```bash
-cargo add leptos --features=csr,nightly # or just csr if you're using stable Rust
+cargo add leptos --features=csr,nightly
+```
+
+Or you can leave off `nighly` if you're using stable Rust
+```bash
+cargo add leptos --features=csr
```
Comment (g-re-g) [inline docs/book/src/02_getting_started.md:31]: The comment in the previous version breaks copy-paste. If you're not familiar with the command line it's no clear why its broken.

File: docs/book/src/02_getting_started.md [hunk 3]
```diff
@@ -37,7 +42,7 @@ cargo add leptos --features=csr,nightly # or just csr if you're using stable Rus
 >
 > If you’d rather use stable Rust with Leptos, you can do that too. In the guide and examples, you’ll just use the [`ReadSignal::get()`](https://docs.rs/leptos/latest/leptos/struct.ReadSignal.html#impl-SignalGet%3CT%3E-for-ReadSignal%3CT%3E) and [`WriteSignal::set()`](https://docs.rs/leptos/latest/leptos/struct.WriteSignal.html#impl-SignalGet%3CT%3E-for-ReadSignal%3CT%3E) methods instead of calling signal getters and setters as functions.
 
-Make sure you've added the `wasm32-unknown-unknown` target do that Rust can compile your code to WebAssembly to run in the browser. 
+Make sure you've added the `wasm32-unknown-unknown` target so that Rust can compile your code to WebAssembly to run in the browser.
```
Comment (g-re-g) [inline docs/book/src/02_getting_started.md:45]: small typo and removed trailing whitespace

**Comment by gbj** [comment]
Looks good, and thanks for the thoughtful comments so I knew what the point of each change was!

**Pull request merged**